package.json

```diff
@@ -268,5 +268,8 @@
   },
   "_moduleAliases": {
     "puppeteer": "node_modules/puppeteer-core"
+  },
+  "engines": {
+    "node": ">=10 <11"
```

Comment:
Maybe with should lockdown the version to 10.16.0 and change everywhere?

Reply:
I think that will make local dev messy if you constantly get warning because you need a specific patch release of node.